spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,20 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'it locks the given account' do
+      other_user = FactoryGirl.create :user
+      post :lock_account, id: other_user.id
+      expect(other_user.reload.access_locked?).to be_truthy
+    end
+  end
+
+  describe '#unlock_account' do
+    it 'it unlocks the given account' do
+      other_user = FactoryGirl.create :user
+      post :unlock_account, id: other_user.id
+      expect(not(other_user.reload.access_locked?)).to be_truthy
```

Comment:
The main part is http://www.rubydoc.info/gems/rspec-expectations/RSpec/Matchers but things like `rspec-rails` add more matchers.